test/Inputs/clang-importer-sdk/usr/include/ObjCConcurrency.h

```diff
@@ -86,7 +85,7 @@ typedef void (^CompletionHandler)(NSString * _Nullable, NSString * _Nullable_res
 -(void)asyncImportSame:(NSString *)operation completionHandler:(void (^)(NSInteger))handler;
 -(void)asyncImportSame:(NSString *)operation replyTo:(void (^)(NSInteger))handler __attribute__((swift_async(none)));
 
--(void)overridableButRunsOnMainThreadWithCompletionHandler:(MAIN_ACTOR_UNSAFE void (^ _Nullable)(NSString *))completion;
+-(void)overridableButRunsOnMainThreadWithCompletionHandler:(MAIN_ACTOR void (^ _Nullable)(NSString *))completion __attribute__((swift_attr("@_predatesConcurrency")));
```

Comment:
The draft proposal specifies that every imported Clang declaration is marked as predating concurrency, so this probably shouldn't be necessary, but it's fine for now.

Reply:
Oh, thanks for pointing this out! I'll do the implied `@_predatesConcurrency` for Objective-C in a follow-up PR.